modules/installing-ocp-agent-ibm-z-kvm.adoc

```diff
@@ -72,6 +74,43 @@ $ virt-install
 <1> For the `--cdrom` parameter, specify the location of the ISO image on the HTTP or HTTPS server.
 endif::pxe-boot[]
 
+. Optional: Enable FIPS mode
++
+If you want to enable FIPS mode on {ibm-z-name} clusters with {op-system-base} KVM you must use PXE boot instead and run the `virt-install` command with the following parameters:
```

Comment:
In this case slightly stronger suggestion to remove "if you want", since this is a procedural step and the "wanting" part is already implied by the user doing the step marked as optional IMO:
```suggestion
To enable FIPS mode on {ibm-z-name} clusters with {op-system-base} KVM you must use PXE boot instead and run the `virt-install` command with the following parameters:
```